.rspec

```diff
@@ -1,3 +1,2 @@
 --color
 --require spec_helper
---backtrace
```

Comment:
Should we add ```.rspec``` to the ignore file? I defo need the backtrace when doing any rspec testing